src/components/EpisodePreview/EpisodePreview.jsx

```diff
@@ -91,18 +91,26 @@ export default function EpisodePreview(props){
 	const { src: nextTrailerSrc } = nextTrailer;
 
 	return (
-		<aside>
-			<a
-				href={ trailerSrc }
-				aria-controls={ previewId }
-				aria-label={ `Watch preview for episode ${episodeNo}: ${title}.` }
-				onClick={ openOverlay }
-			>
-				<img
-					src={ thumbSrc }
-					alt={ thumbDesc }
-				/>
-			</a>
+		<aside className={ s.container }>
+			<div className={ s.thumbnail }>
+				<a
+					href={ trailerSrc }
+					aria-controls={ previewId }
+					aria-label={ `Watch preview for episode ${episodeNo}: ${title}.` }
+					onClick={ openOverlay }
+				>
+					<img
+						src={ thumbSrc }
+						alt={ thumbDesc }
+					/>
+					<svg className={ s.playButton } 
+						 viewBox="0 0 200 200" 
+						 alt="Play video">
+	                    <circle cx="100" cy="100" r="90" fill="none" strokeWidth="15" stroke="#fff"/>\
+	                    <polygon points="70, 55 70, 145 145, 100" fill="#fff"/>\
+                	</svg>
```

Comment:
Good use of SVG, but they tread this weird line of content / asset - so what I tend to do with these is put them into `SHARED/icons/` and do the usual interface stuff w/index.js :

```
import Play from './Play.jsx';
import Stop from './Stop.jsx';

export { Play, Stop };
```

As long as you make it so that you can use them by doing something like:

```
import { Play as PlayIcon } from "SHARED/icons/";

return (
    <PlayIcon />
);
```
and maybe add support to change their colour as a prop?
```
<PlayIcon color="#ff0000" />
```

Reply:
Done!